packages/flutter/lib/src/widgets/bottom_navigation_bar_item.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+import 'package:flutter/foundation.dart';
+
+import 'framework.dart';
+
+/// An interactive button within either material's [BottomNavigationBar]
+/// or the iOS themed [CupertinoTabBar] with an icon and title.
+///
+/// Should not be used on its own but embedded in one of the bottom navigation
```

Comment:
rather than this paragraph I would say something like "This class is rarely used in isolation".